pom.xml

```diff
@@ -14,7 +15,7 @@
     <packaging>hpi</packaging>
     <properties>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-        <jenkins.version>2.223</jenkins.version>
+        <jenkins.version>2.252</jenkins.version>
```

Comment:
This looks like a very aggressive baseline choice. Users on the LTS line that was first released earlier this week won't be able to use the plugin. Unless you're using very recent features, this has drawbacks without benefits. See https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ for our general recommendation here.